src/lib/__tests__/InstantSearch-test.js

```diff
@@ -349,7 +350,7 @@ describe('start', () => {
     search.start();
 
     expect(algoliasearchHelper).toHaveBeenCalledTimes(2);
-    expect(algoliasearchHelper).toHaveBeenCalledWith(searchClient);
+    expect(algoliasearchHelper).toHaveBeenCalledWith(searchClient, indexName);
```

Comment:
Since we do not need this index (it has worked without), do you actually need to check that it's passed?

Reply:
The assertion fails if not all arguments are listed

Reply:
Would using `expect.anything()` work in both cases?